core/shared/src/main/scala/scalaz/zio/stream/StreamPureR.scala

```diff
@@ -18,50 +18,50 @@ package scalaz.zio.stream
 
 import scalaz.zio._
 
-private[stream] trait StreamPure[-R, +A] extends Stream[R, Nothing, A] { self =>
+private[stream] trait StreamPureR[-R, +A] extends ZStream[R, Nothing, A] { self =>
```

Comment:
This should be the `Z` convention too, right? Or maybe we don't need to change it since it's not a user-visible type.

Reply:
I realize that we can actually just drop `R`. `StreamPure`'s entire point is to be free of effects.